bluesky/plans.py

```diff
@@ -1329,9 +1329,9 @@ def rel_spiral(detectors, x_motor, y_motor, x_range, y_range, dr, nth,
     return (yield from spiral(detectors, x_motor, y_motor,
                               x_motor.position, y_motor.position,
                               x_range, y_range, dr, nth, tilt=tilt,
-                              per_step=per_step, md=_md))
-
-
+                              per_step=per_step, md=_md))  
+  
+  
```

Comment:
These are odd blank spaces... Should not be there since the file is not touched.

Reply:
So if you look at the difference there is none between the old and new file, not sure why they are still included as 'changed lines' or how to fix this.